src/pages/billing/CreateAzureBillingProjectModal.test.js

```diff
@@ -4,20 +4,17 @@ import { axe } from 'jest-axe'
 import _ from 'lodash/fp'
 import { act } from 'react-dom/test-utils'
 import { h } from 'react-hyperscript-helpers'
+import { mockModalModule } from 'src/components/Modal.mock'
 import { Ajax } from 'src/libs/ajax'
 import CreateAzureBillingProjectModal from 'src/pages/billing/CreateAzureBillingProjectModal'
 import { billingProjectNameValidator } from 'src/pages/billing/List'
 import { v4 as uuid } from 'uuid'
 
 
 jest.mock('src/components/Modal', () => {
```

Comment:
while this could have been written a bit shorter as:
`jest.mock('src/components/Modal', mockModalModule)`

... I kept the arrow-function format since it is closer to the Typescript recommended pattern I am recommending:
```
jest.mock('src/components/Modal'. (): ModalExports => {
  return mockModalModule()
}
```

which has some added type safety, including if there's a change that jest.mock's loose types would not guard against, especially if further tweaks are layered on. i.e.
```
   return ({
     ...mockModalModule(),
     overriddenExport: somethingInteresting
   })
```